ibm_mq/requirements.in

```diff
@@ -1 +1 @@
-pymqi==1.8.0; sys_platform != 'win32' and sys_platform != 'darwin'
+pymqi==1.8.0; sys_platform != 'win32'
```

Comment:
Merging this may break on our Agent macOS builders, just fyi

Reply:
Good catch, should I just leave it be and add instructions to edit this file when developing on a mac?

Reply:
I think so

Reply:
Done